spec/mas/cms/entity/footer_spec.rb

```diff
@@ -27,7 +27,7 @@
       expect(subject.contact).to be_a(Mas::Cms::Contact)
     end
 
-    it "doesn't make multiple instances of contact if called multiple times" do
```

Comment:
Hey @gotbadger is not better to describe what's going on rather than using the word _memoized_? 

i. `it 'calls are memoized' do` doesn't yield an immediate meaning of what's being done.
ii. the full worded description was better. Do you believe that it missed the point of the test, hence warrants the change?

Reply:
@Tr4pSt3R Memoization is the technical term for whats happening here. I was actually confused by by the description so in my opinion this was a better description of what was being tested. 

Since the test I was writing for `web_chat` is similar to this one I updated it to be consistent. Im not that precious about it so can both to be the old message if you want?

Reply:
We try not to use technical terms in the test descriptions because designers and stakeholders may not be familiar with them. Could you try to think of another description that isn't confusing to you and doesn't use technical terms?

Reply:
@margOnline maybe in a feature spec i could see the reason for this but feels out of place in this kinda of test and repo. The previous message its really quite technical anyway too... 

Regardless I don't think a PR is the place to debate this and have updated the test description to the previous message as a better one was not forthcoming 😄